.gitignore

```diff
@@ -1,8 +1,9 @@
 deps/
 
-cgc
-fuzz
-imgtrace
-repl
-shellcode
-usercorn
+/cgc
```

Comment:
The / ensures it's only these bins at the root of the repo: I noticed that it wouldn't recognize my changes to _go/cmd/com/..._